add-ons/fabric8/fabric8.addon

```diff
@@ -0,0 +1,37 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+# Required-Vars: GITHUB_OAUTH_CLIENT_ID,GITHUB_OAUTH_CLIENT_SECRET
```

Comment:
These are likely only needed when used from a 'crowded' network, where people share an IP address or range. As this sounds much like the issue with downloading the `oc`  client from the Red Hat network in Raleigh.